plugins/QuickScreenshot/main.lua

```diff
@@ -70,8 +71,10 @@ function go()
   -- Different operating systems have different
   -- screenshot utilities, so we check the OS here
   if operatingSystem == "Windows" then
+    app.msgbox("Windows not supported yet.", {})
     print("Windows not supported yet.")
   elseif operatingSystem == "Darwin" then
+    app.msgbox("macOS not supported yet.", {})
```

Comment:
Same here.

Reply:
I do not think printing the error message to the console is necessary if the same message is already communicated via the msgbox. But that is a minor thing.

Reply:
Fixed!